src/config.ts

```diff
@@ -80,13 +80,17 @@ export const connectionOptions = (() => {
 
   convertNamedPlaceholders = options.namedPlaceholders === 'false' ? null : require('named-placeholders')();
 
-  for (const key in ['dateStrings', 'flags', 'ssl']) {
+  for (const key of ['dateStrings', 'flags', 'ssl']) {
     const value = options[key];
 
     if (typeof value === 'string') {
       try {
         options[key] = JSON.parse(value);
-      } catch {}
+      } catch (err) {
+        console.log(
+          `^3Unable to parse ${key} configuration (${err})!^0`
```

Comment:
```suggestion
          `^3Failed to parse property '${key}' in configuration (${err})!^0`
```

Reply:
Super nitpicky on my end, but other than that it looks good. Nice catch.

Reply:
Reworded as per notice and pushed changes :)